projects/angular/src/forms/styles/_properties.forms.scss

```diff
@@ -96,6 +96,26 @@
       // Range
       --clr-forms-range-progress-fill-color: var(--clr-color-action-600);
       --clr-forms-range-track-color: var(--clr-color-neutral-200);
+
+      // Password
+      --clr-forms-password-input-height: #{forms-variables.$clr-forms-password-input-height};
+      --clr-forms-password-padding-bottom: #{forms-variables.$clr-forms-password-padding-bottom};
+      --clr-forms-password-layout-space: #{forms-variables.$clr-forms-password-layout-space};
+      --clr-forms-password-eye-icon-min-height: #{forms-variables.$clr-forms-password-eye-icon-min-height};
+      --clr-forms-password-eye-icon-width: #{forms-variables.$clr-forms-password-eye-icon-width};
+
+      --clr-forms-password-default-typography-color: #{forms-variables.$clr-forms-password-default-typography-color};
+      --clr-forms-password-default-underline-color: #{forms-variables.$clr-forms-password-default-underline-color};
+      --clr-forms-password-active-underline-color: var(--clr-forms-focused-color);
+
+      --clr-forms-password-default-eye-icon-color: #{forms-variables.$clr-forms-password-default-eye-icon-color};
+      --clr-forms-password-disabled-eye-icon-color: #{forms-variables.$clr-forms-password-disabled-eye-icon-color};
+
+      --clr-forms-password-disabled-typography-color: var(--clr-forms-text-disabled-color);
+      --clr-forms-password-disabled-underline-color: var(--clr-forms-text-disabled-color);
+
+      --clr-forms-password-error-underline-color: var(--clr-forms-invalid-color);
+      --clr-forms-password-success-underline-color: var(--clr-forms-valid-color);
```

Comment:
Aren't all these coming from the inputs so the input migration would be enough?

Reply:
But I think we need to allow making custom styling for just the bottom border on success/error of the password (while leaving the other inputs, say datalist with the default color)